### ISSUE TITLE
Ranges try (and fail) to capture both extent and hierarchy

### ISSUE THREAD

**Issue by azaroth42** [IIIF/api#1070]
The canvases / ranges separation allows for ranges to be used to capture navigation hierarchy, separate from the complete extent of the parent range.  2.1 allows the use of members to capture a single order across both canvases and ranges (#716). However, the v3 proposal to reduce down to just members loses the functionality of separation of extent and hierarchy.

For example, a video starts with 20 seconds of applause and audience noise. The start of Act 2 of the Opera then starts at 20 seconds and continues until 620 seconds.  At 60 seconds in, the first song within Act 2 starts for 120 seconds.  Then at 300 seconds in, the second song starts and runs until 360 seconds.  We do not want to create an artificial range for the time spans 0,20 or 180,300, or 360,620.

Extent of: 
 * Act 2 is 20,620
 * Act 2, song 1 is 60,180
 * Act 2, song 2 is 300,360

Goal is to display this hierarchy without additional ranges.

Opera is:
 * Act 1
 * Act 2

**Comment by azaroth42** [comment]
To try and summarize the consensus from the AV discussions, and then relayed on a broader IIIF technical community call...

* Each range will capture its extent, either by including the canvases (or parts thereof) directly or by including further ranges. If a canvas is not transitively included within a range, then it is not part of the range.
* Only ranges, and only ranges with labels, will be present in the navigation displayed to the user.  Other ranges without labels might exist. If a part of a canvas should have a label for navigation, then it must be within a Range.
* Ranges with labels and with a `no-nav` viewingHint will not be displayed in navigation to the user.
* All child ranges and canvases will be included in the `members` property.
* `canvases` and `ranges` properties will be removed from the model.

Example structure:
  https://github.com/zimeon/iiif-ranges-examples/blob/master/book/book_simple_ext3.json

**Comment by azaroth42** [comment]
References:
* Call for participation: https://groups.google.com/forum/?hl=en#!topic/iiif-discuss/V6CewMKWqo4 
* Main AV discussion: https://docs.google.com/document/d/1yKvQrzNcGxbTN2vBlyxX3sG-8_ACy7VfhxtswnVIha0/edit
* Confirmation following experiments: https://docs.google.com/document/d/18gQBHmaANFDxT6AuCaBXwSnasd71iOPMIPAdp57cdkA/edit

**Comment by tomcrane** [comment]
For quick access on call if necessary:

before: http://dams.llgc.org.uk/iiif/newspaper/issue/3082288/manifest.json
after: https://raw.githubusercontent.com/glenrobson/iiif_stuff/master/examples/structures_opt3/ExampleNewspaperIssue.json

**Comment by tomcrane** [comment]
Option 3 agreed on as the solution on AV call 25/7/2017

**Comment by zimeon** [comment]
Has been agreed in previous discussions but will air for final ratification in Toronto (will not discuss in detail as that would take too long).

**Comment by jronallo** [comment]
@tomcrane Where would we find "Option 3"? I'm not sure I'm following which option that is.

**Comment by tomcrane** [comment]
https://github.com/zimeon/iiif-ranges-examples/blob/master/book/book_simple_ext3.json

Sorry not very clear in the comment history. Several variants (in that repo) were put forward.

**Comment by jpstroop** [comment]
Toronto WG Plenary: 👍 after a fair bit of explanation

**Comment by azaroth42** [comment]
Closed by #1309
